### ISSUE TITLE
Please can you help me for subcription with coingate

### ISSUE THREAD

**Issue by Showkiip** [coingate/coingate-php#19]
Hello,
Please help me to integrate coin gate subscriptions with laravel application I have tried the given curl function code by coin gate but it is not working.
Do you help me with how can I do subscriptions with coin gate?
thanks

**Comment by vgumonis** [comment]
Hi, 
what is the issue you are running into ?

**Comment by Showkiip** [comment]
when i call subscription api  this( curl --location --request POST '
https://sandbox.coingate.com/api/v2/billing/details' \) an through
exception 404 not found
i am using apis from there
https://developer.coingate.com/docs/create-billing-details

**Comment by vgumonis** [comment]
could you share the POST body, please remove substitute any sensitive info

**Comment by Showkiip** [comment]
Please check now.  attachment

**Comment by Showkiip** [comment]
post body

**Comment by vgumonis** [comment]
cannot access it

**Comment by Showkiip** [comment]
yes

**Comment by Showkiip** [comment]
showing this error

**Comment by Showkiip** [comment]
do you have any other way to do this.?

**Comment by vgumonis** [comment]
just copy paste the request please

**Comment by Showkiip** [comment]
this is not working

**Comment by Showkiip** [comment]
i have do copy paste but not working

**Comment by Showkiip** [comment]
yes I have just copied paste but not working and no anyone integrates
subscriptions using coin gate and has no help found in google.

**Comment by Showkiip** [comment]
Please can you share the subscription code also like this single payment code?

**Comment by Showkiip** [comment]
this is my code i have convert curl into php
